src/components/Movies.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import "../App.css";
+
+
+class Movies extends React.Component {
+    constructor() {
+        super()
+
+        this.state = {
+            movie: [],
+            movieTitle: null,
+            movieDate: null,
+            movieDescrip: null
+        }
```

Comment:
```suggestion
        this.state = {
            movie: [],
            movieTitle: null
        }
```
You're not using `movieDate` or `movieDescrip` anywhere. You don't need them, you just need to keep track of which `movieTitle` is currently selected.